app/templates/show-project.hbs

```diff
@@ -76,35 +76,46 @@
             <ul class="actions-list no-bullet">
               {{#each model.actions as |action|}}
                 <li class="grid-x grid-padding-small action">
-                  <div class="cell shrink">
+                  <div class="cell shrink statuscode {{dasherize action.statuscode}}">
```

Comment:
is this for CSS or test selectors?

Reply:
CSS, I believe.

Reply:
It's used for coloring them appropriately with CSS.